tests/data/unrecognized-config-fields.toml

```diff
@@ -0,0 +1,66 @@
+[lift]
+name = "igs"
+description = "Test interpreter group selection."
+
+# N.B: `scie-jump` is invalid, should be `scie_jump`.
+[lift.scie-jump]
+version = "0.11.0"
+
+[lift.scie_jump]
+# N.B: `version2` is invalid, should be `version`.
+version2 = "0.11.0"
+
+[[lift.interpreters]]
+id = "cpython310"
+provider = "PythonBuildStandalone"
+version = "3.10"
+lazy = true
+
+[[lift.interpreters]]
+id = "cpython311"
+provider = "PythonBuildStandalone"
+version = "3.11"
+# N.B: `lizzy` is invalid, should be `lazy`.
+lizzy = true
+
+[[lift.interpreter_groups]]
+id = "cpython"
+selector = "{scie.env.PYTHON}"
+members = [
+    "cpython310",
+    "cpython311",
+]
+
+[[lift.commands]]
+# N.B: `environ` is invalid.
```

Comment:
all the other notes state what it should be, so for consistency:
```suggestion
# N.B: `environ` is invalid, should be `env`.
```

Reply:
I actually meant this one since environ does not correspond to anything. The `env` construct has 4 variants 2 replace styles, 2 remove styles. So there is no exact "should be" - totally ambiguous. In this case it's just a bad key plain and simple.

Reply:
As a side note to this one, I was contemplating did-you-mean help, but deferred for later. When adding did you mean help, this sort of thing will be problematic to be sure. It may be best to just stick with the current manifest format doc link and let the human actually read. They should be good at that.